src/site/assets/js/vendor/uswds.min.js

```diff
@@ -1,2 +0,0 @@
-//# sourceMappingURL=uswds.min.js.map
```

Comment:
This is included in the HTML snippet we have inserted into TeamSite pages. See -
```
<script type="text/javascript" src="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/js/vendor/uswds.min.js"></script>
```

In the page source for benefits.va.gov. Removing it will cause that to 404, but it doesn't seem to be used anyway, so I'm not sure what effect that could have.
 

Reply:
How strange.  I suspect that there's a better way of getting that, like from a CDN maybe.